carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -232,13 +232,25 @@ void WMBroadcaster::addPassingControlLineFromMsg(std::shared_ptr<Geofence> gf_pt
 
 void WMBroadcaster::addRegionAccessRule(std::shared_ptr<Geofence> gf_ptr, const cav_msgs::TrafficControlMessageV01& msg_v01, const std::vector<lanelet::Lanelet>& affected_llts) const
 {
-   gf_ptr->regulatory_element_ = std::make_shared<lanelet::RegionAccessRule>(lanelet::RegionAccessRule::buildData(lanelet::utils::getId(),affected_llts,{},participantsChecker(msg_v01)));
+  auto regulatory_element = std::make_shared<lanelet::RegionAccessRule>(lanelet::RegionAccessRule::buildData(lanelet::utils::getId(),affected_llts,{},participantsChecker(msg_v01)));
+
+  if(regulatory_element->accessable("vehicle:car")) //TODO: region_access rule by default blocks everything except participant
```

Comment:
This says participant "vehicle:car" but what I suggested was "participant:vehicle". 
Just to be safe you can just use lanelet::Participants::Vehicle